From 026f4aef871f18f418c77561e27b077ea5fd941f Mon Sep 17 00:00:00 2001
From: Justin Schneck <jschneck@mac.com>
Date: Wed, 17 Aug 2022 18:38:31 -0400
Subject: [PATCH] Do not modify BUILD_PLAT for ti k3

---
 plat/ti/k3/platform.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/ti/k3/platform.mk b/plat/ti/k3/platform.mk
index 2de21aa..825a015 100644
--- a/plat/ti/k3/platform.mk
+++ b/plat/ti/k3/platform.mk
@@ -11,4 +11,4 @@ include ${PLAT_PATH}/common/plat_common.mk
 include ${PLAT_PATH}/board/${TARGET_BOARD}/board.mk
 
 # modify BUILD_PLAT to point to board specific build directory
-BUILD_PLAT := $(abspath ${BUILD_BASE})/${PLAT}/${TARGET_BOARD}/${BUILD_TYPE}
+#BUILD_PLAT := $(abspath ${BUILD_BASE})/${PLAT}/${TARGET_BOARD}/${BUILD_TYPE}
-- 
2.34.1

